src/app/hearing-part/components/listing-create/listing-create.component.spec.ts

```diff
@@ -49,10 +49,8 @@ let storeSpy: jasmine.Spy;
 let component: ListingCreateComponent;
 let store: Store<State>;
 let fixture: ComponentFixture<ListingCreateComponent>;
-let listingCreateNoteConfig: ListingCreateNotesConfiguration;
```

Comment:
Why are you removing this and other notes related elements?

Reply:
notes were passed to listing request effects and from there haven't used anymore. I just removed unnecessary code.